EIPS/eip-1.md

```diff
@@ -35,6 +35,13 @@ It is highly recommended that a single EIP contain a single key proposal or new
 
 An EIP must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate the protocol unduly.
 
+### Special requirements for Core EIPs
+
+If a **Core** EIPs changes the EVM (Ethereum Virtual Machine), it should refer to the instructions by their mnemonics and define the opcodes of those mnemonics at least once. A preferred way is the following:
```

Comment:
EIPs -> EIP

Suggestion for some rephrasing: "...mentions or proposes changes to instructions of the EVM"

Reply:
Thanks! Updated.